.github/ISSUE_TEMPLATE/bug-report.md

```
@@ -0,0 +1,46 @@
+
+## Actual Behavior
+
+## Environment
+
+- Branch / commit:
+- OS / browser / Go / Python / Bun version:
+- Deployment target (local / staging / production):
+
+## Logs / Screenshots
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)